packages/http/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @turnkey/http
 
+## 2.11.0
+
+### Minor Changes
+
+- Update to use new endpoints. Including CREATE_READ_WRITE_SESSION and CREATE_API_KEYS_V2
```

Comment:
Can we add some more color here? For `CREATE_API_KEYS_V2`, maybe something mentioning the addition of `SECP256K1` keys as a valid authenticator

And for `CREATE_READ_WRITE_SESSION`, a short description of what it enables.

Ideally we can use this changelog as a historical record of the new functionality we add on a release-to-release basis (and one day, it would be super dope to have a full fledged changelog type blog a la https://resources.arc.net/hc/en-us/articles/20498293324823-Arc-for-macOS-2024-Release-Notes)

Reply:
100p